src/DotNet.Status.Web/Controllers/AnnotationsController.cs

```diff
@@ -0,0 +1,109 @@
+using Azure.Data.Tables;
+using DotNet.Status.Web.Models;
+using DotNet.Status.Web.Options;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.DotNet.Services.Utility;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Options;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Runtime.CompilerServices;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace DotNet.Status.Web.Controllers
+{
+    /// <summary>
+    /// This Annotations controller serves queries in a format compatible with the Grafana
+    /// "Simple JSON Datasource". It is used to expose the "Deployments" table data to 
+    /// Grafana, which may then render the information in dashboards.
+    /// </summary>
+    [Route("api/[controller]")]
```

Comment:
Could we just put "annotations" here? The magic `[controller]` thing leads to really unexpected breaking behavior when renaming things later.